src/status_im/ui/screens/desktop/main/chat/views.cljs

```diff
@@ -131,9 +151,10 @@
              ^{:key (str message index)}
              [message content (= from @current-public-key) (assoc message-obj :group-chat group-chat)]))]]])))
 
+
 (views/defview chat-text-input []
   (views/letsubs [inp-ref (atom nil)]
-    [react/view {:style {:height 90 :margin-horizontal 16  :background-color :white :border-radius 12}}
+    [react/view {:style {:height 68 :margin-horizontal 16  :background-color :white :border-radius 12}}
```

Comment:
move to styles

Reply:
@flexsurfer, thanks for valuable comments! I'm still only learning how to write clojurescript properly :)
Do you use any code formatter tool for clojurescript? In `react-native-desktop` we use `clang-format` that reformats code by a set of predefined rules automatically on save.

Reply:
that's cool actually you're doing well :) i'm using my IDE for formatting :)